typewritten.zsh

```diff
@@ -31,6 +31,10 @@ _set_left_prompt() {
   local typewritten_prompt="$virtualenv$return_code$prompt_color$prompt_symbol %F{default}"
   local user_host="%F{yellow}%n%F{default}@%F{yellow}%m "
 
+  if [ "$TYPEWRITTEN_DISABLE_RETURN_CODE" == true ]; then
+    return_code = '';
+  fi
+
```

Comment:
Since the `typewritten_prompt` variable has already been defined with the value of `return_code`, the damage would already be done. I suggest moving this up on line 30, just before defining `typewritten_prompt`.